teamB_potentialfunc.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Fri Nov  4 19:49:59 2022
+
+@author: anusheh
+"""
+import numpy as np
+import matplotlib.pyplot as plt
+class Schrodinger:   
+
+    def potential(x):
+        '''
+        function to define the potential this has some built in potentials for commonly used scenarios
+        as well as having the option to enter a custom option
+
+        function takes in (x)
+
+        built in options include for potential:
+        1D_SHO
+        0_potential
+        square_well
+        potential_step
+        '''
+
+        print ('For the potentials there are a few built in options: 1D_SHO, 0_potential, square_well, potential_step or you can choose to enter a custom potential')
+        v_x_selected = input('enter potential V(x): ')
```

Comment:
tell the user how to input the answer. Because if they put "simple harmonic oscillator 1D" for example, the code wouldn't like it

Reply:
Ignore this, it's already done but didn't see